<?xml version="1.0"?>
<launch>

    <node pkg="serp" type="main_node" name="gui" output="screen">
    </node>

    <!-- <node pkg="serp" type="vision_node.py" name="vision" output="screen">
    </node> -->

    <node pkg="serp" type="logic_node" name="logic" output="screen">
    </node>


</launch>